src/main/java/com/philips/swcoe/cerberus/cerebellum/codemetrics/java/CodeMetricsDiffService.java

```diff
@@ -19,15 +19,16 @@
 
     private String previousPath;
     private String currentPath;
+    private CodeMetricsDiffServiceHelper codeMetricsDiffServiceHelper;
 
     public CodeMetricsDiffService(String previousPath, String currentPath) {
         this.previousPath = previousPath;
         this.currentPath = currentPath;
+        this.codeMetricsDiffServiceHelper = new CodeMetricsDiffServiceHelper();
     }
 
-    public List<CodeMetricsClassResult> getMetricsFromSourceCode() {
-        List<CodeMetricsClassResult> codeMetricsClassResultsList =
-            new ArrayList<CodeMetricsClassResult>();
```

Comment:
Request to adjust this to what it was, the previous one was looking more readable